crates/sncast/src/lib.rs

```diff
@@ -252,12 +274,7 @@ pub async fn check_class_hash_exists(
     }
 }
 
-fn get_account_from_keystore<'a>(
-    provider: &'a JsonRpcClient<HttpTransport>,
-    chain_id: FieldElement,
-    keystore_path: &Utf8PathBuf,
-    account: &str,
-) -> Result<SingleOwnerAccount<&'a JsonRpcClient<HttpTransport>, LocalWallet>> {
+fn get_account_info_from_keystore(account: &str, keystore_path: &Utf8PathBuf) -> Result<Account> {
```

Comment:
It depends what we mean by "account" 😅 Currently, It returns an `Account` struct, where it previously returned `SingleOwnerAccount` from starknet-rs. Instead of renaming the function, I would prefer to rename the `Account` struct to something like `AccountInfo`/`AccountData`. Wdyt?

Reply:
AccountData makes sense

Reply:
https://github.com/foundry-rs/starknet-foundry/pull/1850/commits/f3f21ed2ec862b8bf4e6f23dee6941cfa6c04644